iptables-wrapper-installer.sh

```diff
@@ -109,12 +109,28 @@ set -eu
 # kubelet) has already created some iptables rules.
 EOF
 
+cat >> "${sbin}/iptables-wrapper" <<EOF
+tmp_file_legacy=\$(mktemp)
```

Comment:
NOTE: at least `mktemp` is available inside kube-proxy image.
but  `sh: mktmp: command not found` inside calico-node image...
The availability  varies between base images.